packages/patternfly-4/react-core/src/components/Alert/Alert.js

```diff
@@ -60,19 +64,31 @@ const Alert = ({
     </React.Fragment>
   );
 
-  const customClassName = css(styles.alert, getModifier(styles, variant, styles.modifiers.info), className);
-
+  const customClassName = css(
+    styles.alert,
+    getModifier(styles, variant, styles.modifiers.info),
+    className,
+    (toast && 'pf-m-live')
+  );
   return (
     <div {...props} className={customClassName} aria-label={ariaLabel}>
       <AlertIcon variant={variant} />
-      <h4 className={css(styles.alertTitle)}>{readerTitle}</h4>
-      {children && (
-        <div className={css(styles.alertDescription)}>
-          <p>{children}</p>
-        </div>
-      )}
+      <div
+        aria-live={toast ? 'polite' : undefined}
+        aria-relevant={toast ? 'additions text' : undefined}
+        aria-atomic={toast ? 'false' : undefined}
+        className="pf-c-alert__content">
```

Comment:
is there `styles.alertContent`?

Reply:
I don't believe so, similary to `pf-m-live`, I think this was added recently.

<img width="686" alt="Screen Shot 2019-04-15 at 10 00 55 AM" src="https://user-images.githubusercontent.com/5942899/56155987-30c73d00-5f8a-11e9-94bf-b4033a38cb7c.png">